built-ins/authoring-built-ins.md

```diff
@@ -0,0 +1,132 @@
+> ⚠️ Built-in Template Specs are a feature in development and are currently not available for public use. This document describes the initial process required for built-in template spec authoring, but will remain incomplete (contains some placeholders) until the feature has shipped.
+
+
+ **IMPORTANT:** Authoring and contribution of Built-in Template Specs is restricted to **authorized collaborators only** such as internal Microsoft teams and partners. PRs from unauthorized individuals will be rejected with no exceptions.
+
+# Authoring and Contributing Built-in Template Specs
+
+This documents provides general information on what built-in template specs are, and the process involved to contribute new built-in template specs for consumption by a wider population.
+
+## What are built-ins?
+Built-in template specs are [template specs](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-specs?tabs=azure-powershell) that deploy cloud components/settings that fulfill scenarios that are seen as useful for a wide variety of customers across Azure. Unlike private template specs which are only available to customers (tenants) who created them, built-in template specs are available to all customers in a specific cloud (such as the Public cloud, or the Chinese national cloud, etc. etc.). Essentially they are "global" template specs that any customer can deploy from any region, and they'll behave in the same way for each customer.
+
+They share all the same schemas and properties as customer created template specs (including versions), except built-ins are read-only resources that are accessible to anyone (there is no need to manage access rights to built-in template specs).
+
+Examples of useful built-ins include those that apply certain standards/policies (such as ISO 27001) to existing resources, or built-ins that simplify common deployment scenarios (like deploying a Windows VM).
+
+## How do I contribute a new built-in?
+
+Before continuing it's important to determine whether or not the scenario the underlying Azure Resource Manager template fulfills meets the guidelines for being a built-in template spec. Ask yourself:
+
+ - Is it a scenario that will be commonly used by many customers across Azure without requiring tweaks to the template itself?
+ - Is it a scenario that works well "out of the box" (i.e. without a very specific complex environment already setup)?
+
+If the answer to both of these questions is yes, you're on your way towards contributing a built-in. For a new built-in contribution you can follow these steps:
+
+### Step 1: Validate your template/template spec functionality 
```

Comment:
Should we require evidence of this testing in the form of correlation ID of a successful deployment? We could use that to compare template hashes from logs as a basic sanity check.

Reply:
It might be a good fallback if we find it questionable whether someone has actually tested their template. As for whether we'll have the cycles to actually validate each deployment and confirm it's the correct template, I think that's more questionable. I can add correlation ID requirement info to the authoring doc and the PR template, it might cause submitters to be more thorough in their testing.

Reply:
Could just be a CI job that queries the logs and adds a comment with the template hashes for the reviewer to compare. But yeah we can start with collecting the ID first.